lib/i18n/js.rb

```diff
@@ -161,7 +170,7 @@ def self.filter(translations, scopes)
 
     # Initialize and return translations
     def self.translations
-      ::I18n.backend.instance_eval do
```

Comment:
@PikachuEXE Thanks for getting this released so quickly! I just tested it within our app to replace the monkey patch and now it does not load new translations when files are changed in development. This seems to be the case, because Rails calls `I18n.backend.reload!` for each request, so when the translations are fetched again from `I18n::JS.backend`, it still gets the old ones, since this instance has not been reloaded.

I was able to work around this in our case using:
```ruby
I18n::JS.backend = I18n.backend
I18n.backend = I18n::Backend::Chain.new(SomeCustomBackend.new, I18n.backend)
```

Since `Chain#reload!` calls `reload!` on all of its backends, the instance of `I18n::JS.backend` is also reloaded and everything works.

If one does not use a simple backend at all however, this is not an option.
Would it make sense to actually call `reload!` here if the backend is `initialized?`, since it is only called by the sprockets engine when something has changed anyways?

Reply:
Yeah, you are probably right. I added some documentation in #521 with possible workarounds.